R/property.R

```diff
@@ -260,28 +260,16 @@ prop_label <- function(object, name) {
   sprintf("%s@%s", if (!is.null(object)) obj_desc(object) else "", name)
 }
 
-#' @rdname prop
-#' @usage object@name
-#' @aliases @
-#' @rawNamespace if (getRversion() >= "4.3.0") S3method(base::`@`, S7_object) else export("@")
-`@.S7_object` <- prop
-
 # Note: we need to explicitly refer to base with "base::`@`" in the
```

Comment:
I think you'll want to move this comment now

Reply:
This PR moves it to be above the `@rawNamespace`. Do you think it should go elsewhere?

Reply:
Oh ignore me, read it wrong